x-pack/plugins/siem/public/components/page/network/ip_overview/index.tsx

```diff
@@ -40,10 +41,18 @@ interface OwnProps {
 
 export type IpOverviewProps = OwnProps;
 
+const DescriptionList = styled(EuiDescriptionList)`
```

Comment:
I am wondering if we can avoid styled component here by doing that in the props of Description list `titleProps={{ className: 'euiText--extraSmall' }}`

Reply:
I worry that might be a little dangerous, as it applies a class that is meant for a different component altogether. If the Kibana design team were ever to change those styles for the `EuiText` component, it could have adverse affects on our description list. It becomes doubly problematic if/when the Kibana design team moves to CSS modules, which would essentially render this global class useless.

I think the styled component override is the best bet for now, with the possibility of adding it as a prop to the `EuiDescriptionList` in the future (to cut out such overrides).